File: php/create_campaign.php
<?php
// GUZZLE
// Make sure to install Guzzle using Composer: composer require guzzlehttp/guzzle

use GuzzleHttp\Client;
use GuzzleHttp\Psr7\Request;

// Replace the following values with your own
$prodUrl = '<REPLACE_WITH_PROD_URL>';
$token = '<YOUR_TOKEN>';

$client = new Client();
$headers = [
    'Authorization' => 'Bearer ' . $token,
    'Content-Type' => 'application/json'
];
$body = '{
  "label": "My Campaign publipostage",
  "sender": "AROLITEC",
  "contacts": [
    {
      "age": "12",
      "numero": "22507479171",
      "name": "Martial"
    },
    {
      "age": "24",
      "numero": "22507051918",
      "name": "Rita"
    },
    {
      "age": "32",
      "numero": "225068275212",
      "name": "Arolitec"
    }
  ],
  "content": "Hello {{name}}, votre numéro est {{numero}}"
}';
$request = new Request('POST', $prodUrl . '/v1/campaigns', $headers, $body);
$res = $client->sendAsync($request)->wait();
echo $res->getBody();

File: vb.net/create_campaign.vb
Dim client As New RestClient("<REPLACE_WITH_PROD_URL>/v1/campaigns")

client.Timeout = -1

Dim request As New RestRequest(Method.POST)

request.AddHeader("Authorization", "Bearer dc408be00542eed70f4e7d1336d9a38735e5fa74")
request.AddHeader("Content-Type", "application/json")
Dim body As String = "{" & vbCrLf &
"    ""label"": ""My Campaign publipostage""," & vbCrLf &
"    ""sender"": ""SMS INFO""," & vbCrLf &
"    ""contacts"": [" & vbCrLf &
"        {""age"": ""12"", ""numero"": ""225074718911"", ""name"": ""Martial""}," & vbCrLf &
"        {""age"": ""24"", ""numero"": ""225070865198"", ""name"": ""Rita""}," & vbCrLf &
"        {""age"": ""32"", ""numero"": ""225076827512"", ""name"": ""Arolitec""}" & vbCrLf &
"    ]," & vbCrLf &
"    ""content"": ""Hello {{name}}, votre numéro est {{numero}}""" & vbCrLf &
"}"
request.AddParameter("application/json", body, ParameterType.RequestBody)

Dim response As IRestResponse = client.Execute(request)
Console.WriteLine(response.Content)